runtime/src/sdk.rs

```diff
@@ -574,18 +554,10 @@ pub unsafe extern "C" fn extism_plugin_call(
         }
     }
 
-    // Stop timer
-    if let Err(e) = plugin.stop_timer() {
-        let id = plugin.timer_id;
-        return plugin.error(
-            format!("Failed to stop timeout manager for {id}: {e:?}"),
-            -1,
-        );
-    }
-
     match res {
         Ok(()) => (),
         Err(e) => {
+            plugin.store.set_epoch_deadline(1);
```

Comment:
figuring all of this _is_ actually expected to remain here... I suppose this is at minimum related to the Rust SDK then, and possibly does need to be implemented in kernel-based SDKs?

Reply:
Yeah, timeouts and memory limits are features that will have to be implemented by each kernel-based SDK. This PR doesn't really change anything, but it does make it possible to start building up new SDKs that don't rely on this code anymore. Like in Go, cancellation suddenly becomes way easier because of Contexts, but in Rust it's kinds of a pain right now. If we decided to have an async runtime it would probably change the design a little bit - those are all things we should definitely explore!